pkg/controllers/monitoring-stack/controller.go

```diff
@@ -42,12 +48,19 @@ import (
 	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 )
 
+const (
+	grafanaDatasourceOwnerName      = "monitoring-stack-operator/owner-name"
+	grafanaDatasourceOwnerNamespace = "monitoring-stack-operator/owner-namespace"
+)
+
 type reconciler struct {
-	k8sClient             client.Client
-	scheme                *runtime.Scheme
-	logger                logr.Logger
-	instanceSelectorKey   string
-	instanceSelectorValue string
+	k8sClient                  client.Client
+	scheme                     *runtime.Scheme
+	logger                     logr.Logger
+	instanceSelectorKey        string
+	instanceSelectorValue      string
+	shouldCreateGrafanaDSWatch bool
```

Comment:
may want to use the name var name as Filips patch for consistent. [`grafanaWatchEstablished`](https://github.com/rhobs/monitoring-stack-operator/pull/103/files#diff-606d75b2cccce3c13550d55e0c1b27bb1c1fad0431b4f16d466e41dd4e0c801dR69)